src/modules/dashboard/components/ActionsPageFeed/ActionsPageEvent/ActionsPageEvent.tsx

```diff
@@ -334,6 +343,34 @@ const ActionsPageEvent = ({
                   </span>
                 ),
                 storageSlot: values?.slot?.toHexString(),
+                amountTag: (
+                  <Tag
+                    appearance={{
+                      theme: 'primary',
+                      colorSchema: 'inverted',
+                      fontSize: 'tiny',
+                    }}
+                  >
+                    <Numeral
+                      value={values?.stakeAmount || 0}
+                      unit={getTokenDecimalsWithFallback(token?.decimals || 0)}
```

Comment:
don't put `0` as a fallback for decimals!!! That's super bad. 

You're basically trying to calculate _(with 0)_, 10 to the power of 0, which is all sorts of nasty.

**That's why you are using `getTokenDecimalsWithFallback` so you don't have to pass in a fallback!**